app/models/responsible_body.rb

```diff
@@ -232,11 +231,11 @@ def next_school_sorted_ascending_by_name(school)
   end
 
   def orders_managed_centrally?
```

Comment:
The latest findings also throw doubt on the accuracy of this method name.
The method body is only referencing the default in the absence of a per school setting.
Perhaps this method should be checking the `vcap_feature_flag` state or it should be renamed.

Reply:
It was named like that to exactly match the name of the equivalent method on schools.
What about renaming this to: `responsible_body_will_order_devices_for_schools_by_default?`

Reply:
That name sounds good to me.